spacepy/plot/utils.py

```diff
@@ -1400,3 +1401,149 @@ def show_used(fig=None):
                      alpha=0.3, figure=fig, axes=ax, ec='none', fc=next(colors),
                      fill=True)))
     return rects
+
+def add_arrows(lines, n=3, size=12, style='->', dorestrict=False,
+               positions=False):
+    '''
+    Add directional arrows along a plotted line.  Useful for plotting flow
+    lines, magnetic field lines, or other directional traces.
+
+    *lines* can be either :class:`~matplotlib.lines.Line2D`, a list or tuple
+    of lines, or a :class:`~matplotlib.collections.LineCollection` object.
+
+    For each line, arrows will be added using 
+    :method:`~matplotlib.axes.Axes.annotate`.  Arrows will be spread evenly
+    over the line using the number of points in the line as the metric for
+    spacing.  For example, if a line has 120 points and 3 arrows are requested,
+    an arrow will be added at point number 30, 60, and 90.
+    Arrow color and alpha is obtained from the parent line.
+
+    Parameters
+    ==========
+    lines : :class:`~matplotlib.lines.Line2D`, a list/tuple, or :class:`~matplotlib.collections.LineCollection`
+        A single line or group of lines on which to place the arrows.
+        Arrows inherent color and transparency (alpha) from the line on which
+        they are placed.
+
+
+    Other Parameters
+    ================
+    n : integer
+        Number of arrows to add to each line; defaults to 3.
+
+    size : integer
+        The size of the arrows in points.  Defaults to 12.
+
+    style : string
+        Set the style of the arrow, e.g. '->' (default) or '-|>'
```

Comment:
This uses matplotlib arrow styles? This should probably be mentioned explicitly here (it took me a bit to realize), as it'd give the user a clue on where to look for what the available arrow styles are and how they look.

Reply:
Done; push forthcoming.